### ISSUE TITLE
Selected attribute not being removed with `false`

### ISSUE THREAD

**Issue by davidmerrique** [WebReflection/hyperHTML#117]
Hi, I think this is a bug. I'm trying to make the first option in an array of options have both selected and disabled attributes like this:
```javascript
`
<select>${[...Array(10).keys()].map(i => hyperHTML()`
  <option 
    value=${i}
    selected=${i === 0}
    disabled=${i === 0}
  >${i}</option>`
)}</select>
`
```

The first one is `disabled` and all of them end up having `selected`.

Here's a CodePen https://codepen.io/davidmerrique/pen/QqKmdP?editors=0010

Thanks!

**Comment by WebReflection** [comment]
one does not disable options ... but the select. however, there is some weird bug going on ... yet, you disable the select, and you put selected in the option.

**Comment by davidmerrique** [comment]
I do it all the time to use an option like a placeholder
https://stackoverflow.com/a/13492609

**Comment by papas-source** [comment]
Well I guess that in most cases, the `placeholder` option (because this is actually what you are trying to accomplish) is static, and it is not in an array or object, so you would preferably write it this way

```
<select>
    <option class="placeholder" selected disabled value="">Select language</option>${[...Array(10).keys()].map(i => hyperHTML()`
  <option value=${i}>${i}</option>`
)}</select>
```


nevertheless, this is an actual bug :)

**Comment by davidmerrique** [comment]
@papas-source Of course! I'll use that as a workaround for now. Thanks.

It could even have a hidden attribute 
```javascript
`
<select>
  <option selected disabled hidden>Select language</option>
  ${options}
</select>
`
```

**Comment by WebReflection** [comment]
actually, if I select the first option, both `$0.disabled` and `$0.selected` are `true` which is expected.

What you think you have discovered, is just the HTML part as attribute, which is not the real state of the component.

This is intentional left as it is because Custom Elements need attributes there to trigger `attributeChangedCallback` so whatever fix I land to not show you that attribute in `options` will be irrelevant for the layout.

TL;DR an `<input value="test">` don't necessary have `input.value === "test"`, that is just what the HTML shows to you, not the `input` or element state. In few words, your debugging is half performed, everything works actually fine.

**Comment by WebReflection** [comment]
fixed in `v1.9.2`
forked counter-example: https://codepen.io/WebReflection/pen/QqGgXL?editors=0010
